### ISSUE TITLE
The `pygrib.open` object adds support for the built-in `len` function.

### ISSUE THREAD

**Issue by Clarmy** [jswhit/pygrib#239]
Whether it is possible to extend support for the `len` function to the object returned by `pygrib.open`, such that it could be used like this:
```python
import pygrib

grb = pygrib.open('./myfile.grb')
print(len(grb)) 
```

**Comment by akrherz** [comment]
@Clarmy , are you expecting that to return the number of grib messages or the byte size of the grib file?

**Comment by Clarmy** [comment]
@akrherz, I feel that logically, it should correspond to the count of messages; for checking the byte size of the file, in fact, there's no need to use pygrib either.

**Comment by jswhit** [comment]
the `messages` attribute of the `open` instance is the number of messages in the file. It would be pretty straighforward to have `__len__` return this.

**Comment by Clarmy** [comment]
@jswhit Alright, I see your point, but I believe this usage is not straightforward. At least, I wasn't aware of it until you mentioned it. In contrast, the `len` function is a very common and intuitive method that most Python users instinctively reach for when they need to determine the length of a sequence-type object.

**Comment by akrherz** [comment]
@Clarmy , he is just mentioning the class dunder method that makes the `len()` function work.  For example

```python
class MyObj(object):

    def __len__(self):
        return 42


print(len(MyObj()))
```
results in 42 being printed.

**Comment by jswhit** [comment]
This feature is implemented in version 2.1.6
